### PR TITLE
docs: added net failover docs

### PR DESCRIPTION
**Pull request by marcellorinaldo**
N/A.

**Related Issue:** N/A.

**Description of the solution adopted:** N/A.

**Screenshots:** N/A.

**Any side note on the changes made:** N/A.

### CODE REVIEW

File: docs/gateway-configuration/network-configuration.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ Name                                             | Type     | Description
 Name                                                | Type     | Description
 ----------------------------------------------------|----------|------------------------------------------
 `net.interface.<interface>.config.ip4.status`	    | String   | The status of the interface for the IPv4 configuration; possibile values are: netIPv4StatusDisabled, netIPv4StatusUnmanaged, netIPv4StatusL2Only, netIPv4StatusEnabledLAN, netIPv4StatusEnabledWAN, netIPv4StatusUnknown
+`net.interface.<interface>.config.ip4.wan.priority` | Integer | (NetworkManager only) Priority used to determine which interface select as primary WAN. Allowed values range from -1 to 2147483647. See [Network Failover](./network-failover.md) for further details
```
Comment (GregoryIvo) [inline docs/gateway-configuration/network-configuration.md]: Is the range inclusive or exclusive?

File: docs/gateway-configuration/network-failover.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Network Failover
+
```
Comment (GregoryIvo) [inline docs/gateway-configuration/network-failover.md:2]: "and a network failover basic functionality."

and basic network failover functionality? maybe this is better

File: docs/gateway-configuration/network-failover.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+For devices configured to use NetworkManager, it is possible to configure multiple WAN interfaces and a network failover basic functionality.
+
+As in picture below, the Kura UI allows defining multiple WAN interfaces. Each WAN interface can be configured with a **WAN Priority** configuration. This priority is used to determine which interface select as primary WAN. In case the primary WAN interface loses connection, then the next highest priority interface is selected.
```
Comment (GregoryIvo) [inline docs/gateway-configuration/network-failover.md]: "which will be prioritized for Lan"?

wording is a bit AWk here

**Review by GregoryIvo — CHANGES_REQUESTED** [review]
Just some instances of awkward phrasing.  Other then that looks good!

File: docs/gateway-configuration/network-configuration.md (hunk 1 above)
Comment (marcellorinaldo) [reply docs/gateway-configuration/network-configuration.md]: Inclusive

File: docs/gateway-configuration/network-failover.md (hunk 2 above)
Comment (marcellorinaldo) [reply docs/gateway-configuration/network-failover.md:2]: Yes, it is better

**Review by GregoryIvo — APPROVED** [review]

File: docs/gateway-configuration/network-failover.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # Network Failover
 
-For devices configured to use NetworkManager, it is possible to configure multiple WAN interfaces and a network failover basic functionality.
+For devices configured to use NetworkManager, it is possible to configure multiple WAN interfaces and and a basic network failover functionality.
```
Comment (GregoryIvo) [inline docs/gateway-configuration/network-failover.md]: and and

**Review by GregoryIvo — CHANGES_REQUESTED** [review]

**Pull request merged**